omod/src/main/resources/apps/ugandaemr_reports_app.json

```diff
@@ -501,6 +501,30 @@
         "url": "reportingui/runReport.page?reportDefinition=e38117c0-9ddd-4d51-b3b7-a39e38fd85a1",
         "order": 130,
         "requiredPrivilege": "App: ugandaemrreports.transitionedToDTGReport"
+      },{
+        "id": "Patients on First Line ART Regimen",
+        "extensionPointId": "org.openmrs.module.ugandaemr.reports.overview",
+        "type": "link",
+        "label": "Patients on First Line ART Regimen",
+        "url": "reportingui/runReport.page?reportDefinition=2fc0da7b-c297-48b2-857e-113da2a3b357",
+        "order": 135,
+        "requiredPrivilege": "App: ugandaemrreports.patientsOnFirstLineARTRegimen"
+      },{
+        "id": "Patients on Second Line ART Regimen",
+        "extensionPointId": "org.openmrs.module.ugandaemr.reports.overview",
+        "type": "link",
+        "label": "Patients on First Second ART Regimen",
+        "url": "reportingui/runReport.page?reportDefinition=864b1e53-eb4c-4d74-bfbd-331f95554150",
+        "order": 140,
+        "requiredPrivilege": "App: ugandaemrreports.patientsOnSecondLineARTRegimen"
+      },{
+        "id": "Patients on Third Line ART Regimen",
+        "extensionPointId": "org.openmrs.module.ugandaemr.reports.overview",
+        "type": "link",
+        "label": "Patients on First Line ART Regimen",
```

Comment:
Report label is not showing first line instead of third line

Reply:
Done